cartridges/app_komoju_controllers/cartridge/controllers/KomojuController.js

```diff
@@ -552,7 +561,25 @@ function HandleWebHooksCaptureComplete() {
     var body = JSON.parse(request.httpParameterMap.requestBodyAsString);
     var paymentInstrument;
     var orderStatus;
+
+    try {
+        Transaction.wrap(function () {
+            order.custom.komojuPaymentId = paymentIdKomoju;
+        });
+    } catch (e) {
+        Logger.error('Error saving paymentId ' + e.toString() + ' in ' + e.fileName + ':' + e.lineNumber);
```

Comment:
What do you think about noting that this error happens in the `HandleWebHooksCaptureComplete` webhook instead of the filename and line number? I think it may be easier to understand the issue when seeing the log for debugging 🤔